### PR TITLE
Accessibility updates to homepage

### PR DESCRIPTION
**Pull request by nicolesylvester**

### CODE REVIEW

File: lib/home.dart [hunk 1]
```diff
@@ -73,10 +73,23 @@ class _MyHomePageState extends State<MyHomePage> {
...
-                'assets/rrdb_logo.png',
-                height: 55,
-              ),
+              Focus(child: Builder(builder: (context) {
+                final bool hasFocus = Focus.of(context).hasFocus;
+                return Container(
+                  decoration: BoxDecoration(
+                    border: hasFocus
+                        ? Border.all(
+                            // color: Colors.black,
```
Comment (JavadocMD) [inline lib/home.dart:82]: Should this use theme primary color (or does it already by default)?

**Review by JavadocMD — COMMENTED** [review]
This is looking good! See one inline comment.

Other than that, I did a quick Google and found the the Focus widget does have an [onKeyEvent](https://api.flutter.dev/flutter/widgets/Focus/onKeyEvent.html) handler that we might be able to leverage to do "enter to click the link" stuff.

Rather than wrapping Link in a Focus widget, did you consider putting the Focus widget inside Link? (It's a custom class so we can manipulate it how we like.) Of course this may cause other problems, but it would reduce the need to repeat ourselves since we'll probably want every Link to be focusable.

**Review by JavadocMD — APPROVED** [review]
Nice, I see using InkWell simplified things a bit for us. Good find. Question: when hovering over the link now does it still change the mouse cursor to the pointy-hand cursor -- the one you'd normally see when hovering a browser link? (I think this emoji is close -- :point_up_2:)

I just want to make sure that's maintained -- those sorts of interface conventions tend to help users understand what will happen when they click on something.

**Comment by nicolesylvester** [comment]
@JavadocMD Yes, I just checked, and the cursor still changes to a pointy-hand cursor when hovering over the links

**Pull request merged**